CONTRIBUTING.md

```diff
@@ -30,22 +38,22 @@ Ready to contribute? Here's how to set up `aicsimageio` for local development.
     Ex: feature/read-tiff-files or bugfix/handle-file-not-found<br>
     Now you can make your changes locally.
 
-5. When you're done making changes, check that your changes pass linting and
+6. When you're done making changes, check that your changes pass linting and
```

Comment:
I might suggest changing these all to `1.` so the markdown render renumbers them appropriately when steps change

Reply:
Agree.